assets/rich_text_components_definitions.ts

```diff
@@ -145,7 +145,14 @@ export = {
     "is_lesson_related": true,
     "is_block_element": false,
     "customization_arg_specs": [{
-      "name": "skill_summary",
+      "name": "text",
+      "description": "Enter the link text to be displayed",
+      "schema": {
+        "type": "unicode"
+      },
+      "default_value": ""
+    }, {
+      "name": "skill_id",
       "description": "Click on skill to view concept card.",
```

Comment:
Shouldn't this be: "The ID of the skill to reference"?

Reply:
This is actually the text displayed when the element in clicked,  so here:
![image](https://user-images.githubusercontent.com/22347970/75375114-79ae8880-58f3-11ea-9713-fbbf0b82eb69.png)
So, 'ID of skill to reference' wouldn't work here.

Reply:
How about:

- "The link text to be displayed" (strike "Enter")
- "The skill that this link refers to"

?

Reply:
Done